src/main/java/com/minecolonies/core/colony/buildings/AbstractBuilding.java

```diff
@@ -1152,7 +1152,7 @@ public Map<Predicate<ItemStack>, Tuple<Integer, Boolean>> getRequiredItemsAndAmo
     @Override
     public boolean canEat(final ItemStack stack)
     {
-        return stack.getItem().getFoodProperties(stack, null).getNutrition() >= getBuildingLevel();
+        return getBuildingLevel() < 3 || stack.getItem().getFoodProperties(stack, null).getNutrition() >= getBuildingLevel() + 1;
```

Comment:
Since we have so many different food conditions all over the place, how about we move them into their own static util? So we have all the formulas at hand
e.g. static boolean canEat(Building, stack)
FoodItemListModuleWindow contrsuctor has some aswell, food predicate could go there etc